examples/hybrid/sphere/limiters_deformation_flow.jl

```diff
@@ -0,0 +1,428 @@
+    ω = @. ω_0 * sind(λpf) * cosd(ϕf) * cos(2 * pi * t / τ) * sp
+    uw = @. -ω / ρ_ref(zf) / grav
+
+    uₕ = Geometry.Covariant12Vector.(Geometry.UVVector.(uu, uv))
+    w = Geometry.Covariant3Vector.(Geometry.WVector.(uw))
+
+    # Compute velocity by interpolating faces to centers
+    cw = If2c.(w)
+    cuvw = Geometry.Covariant123Vector.(uₕ) .+ Geometry.Covariant123Vector.(cw)
+
+    # Create convenient aliases:
+    ρ = y.ρ
+    # packed version of tracers tuple
+    ρq = y.tracers
+    # unpacked version of tracers
+    @unpack ρq1, ρq2, ρq3, ρq4, ρq5 = y.tracers
+
+    dρ = dydt.ρ
+    # packed version of tracers tuple
+    dρq = dydt.tracers
+    # unpacked version of tracers
+    dρq1 = dydt.tracers.ρq1
+    dρq2 = dydt.tracers.ρq2
+    dρq3 = dydt.tracers.ρq3
+    dρq4 = dydt.tracers.ρq4
+    dρq5 = dydt.tracers.ρq5
+
+    # Define vertical fluxes:
+    # - Density vertical flux
+    vert_flux_wρ = vdivf2c.(w .* Ic2f.(ρ))
+    # - Tracers vertical flux
+    vert_flux_wρq1 = vdivf2c.(Ic2f.(ρ) .* third_order_upwind_c2f.(w, ρq1 ./ ρ),)
+    vert_flux_wρq2 = vdivf2c.(Ic2f.(ρ) .* third_order_upwind_c2f.(w, ρq2 ./ ρ),)
+    vert_flux_wρq3 = vdivf2c.(Ic2f.(ρ) .* third_order_upwind_c2f.(w, ρq3 ./ ρ),)
+    vert_flux_wρq4 = vdivf2c.(Ic2f.(ρ) .* third_order_upwind_c2f.(w, ρq4 ./ ρ),)
+    vert_flux_wρq5 = vdivf2c.(Ic2f.(ρ) .* third_order_upwind_c2f.(w, ρq5 ./ ρ),)
+    # NOTE: When Upwind3rdOrderBiasedProductC2F is extended to work on tuples of tracers, we can use one call for all the vertical tracer fluxes, i.e.:
+    # vert_flux_wρq = vdivf2c.(Ic2f.(ρ) .* third_order_upwind_c2f.(w, ρq ./ ρ),)
+
+    ## NOTE: With limiters, the order of the operations 1)-5) below matters!
+
+    # 1) Horizontal Transport:
+    # 1.1) Horizontal part of continuity equation:
+    @. dρ = beta * dρ - alpha * hdiv(ρ * cuvw)
+
+    # 1.2) Horizontal advection of tracers equations:
+    @. dρq1 = beta * dρq1 - alpha * hdiv(ρq1 * cuvw)
+    @. dρq2 = beta * dρq2 - alpha * hdiv(ρq2 * cuvw)
+    @. dρq3 = beta * dρq3 - alpha * hdiv(ρq3 * cuvw)
+    @. dρq4 = beta * dρq4 - alpha * hdiv(ρq4 * cuvw)
+    @. dρq5 = beta * dρq5 - alpha * hdiv(ρq5 * cuvw)
+
+    # 2) Apply the limiters:
+    if parameters.lim_flag
+        # First compute the min_q/max_q at the current time step
+        Limiters.compute_bounds!(parameters.limiter1, ρq1, ρ)
+        Limiters.compute_bounds!(parameters.limiter2, ρq2, ρ)
+        Limiters.compute_bounds!(parameters.limiter3, ρq3, ρ)
+        Limiters.compute_bounds!(parameters.limiter4, ρq4, ρ)
+        Limiters.compute_bounds!(parameters.limiter5, ρq5, ρ)
+        # Then apply the limiters
+        Limiters.apply_limiter!(dρq1, dρ, parameters.limiter1)
+        Limiters.apply_limiter!(dρq2, dρ, parameters.limiter2)
+        Limiters.apply_limiter!(dρq3, dρ, parameters.limiter3)
+        Limiters.apply_limiter!(dρq4, dρ, parameters.limiter4)
+        Limiters.apply_limiter!(dρq5, dρ, parameters.limiter5)
```

Comment:
This could be simplified by using only one limiter instance, but the current limiters interface would need to be extended to be able to store multiple tuples of q_min/q_max, one for each tracer. I will revisit this in a follow-up PR once this is changed in the library code